rpcserver.go

```diff
@@ -4866,7 +4879,7 @@ func marshalAssetLeaf(ctx context.Context, keys taprpc.KeyLookup,
 	assetLeaf *universe.Leaf) (*unirpc.AssetLeaf, error) {
 
 	rpcAsset, err := taprpc.MarshalAsset(
-		ctx, assetLeaf.Asset, false, true, keys,
+		ctx, assetLeaf.Asset, false, true, keys, fn.None[uint32](),
```

Comment:
This (`marshalAssetLeaf`) is called from multiple places. Should we add the parameter here too and then decide at each call site if it would be possible to get the decimal display or not?
Mainly thinking about `marshalUniverseProofLeaf`, `marshalUniverseDiff` and `AssetLeaves` where the decimal display would be useful to see.

Reply:
Main thing I encountered when coming from universe related code paths is that there's no access to the meta reveal, which seems to require `AssetStore`. Unless there's some other (easier) way to acquire the meta reveal, should I add asset store access to all the subsystems that call this?

Reply:
Hmm, I see. Let's skip it then. But then I think we should keep the `DecimalDisplay` message as `null` on the RPC level. Otherwise it would look like such an asset on the universe RPC side doesn't have a decimal display set, but in reality we just don't know. So maybe, to be able to show the three different states (we don't know if decimal display is set, it is explicitly not set, it is explicitly set), we should pass in the `*taprpc.DecimalDisplay` message here directly. When it's `nil`, we don't know, when it's not `nil` the boolean inside has a meaning.